src/theory/arith/nl/transcendental/transcendental_state.cpp

```diff
@@ -294,12 +294,12 @@ std::pair<Node, Node> TranscendentalState::getClosestSecantPoints(TNode e,
 }
 
 Node TranscendentalState::mkSecantPlane(
-    TNode arg, TNode b, TNode c, TNode approx, TNode approx_c)
+    TNode arg, TNode b, TNode c, TNode approx_b, TNode approx_c)
 {
   NodeManager* nm = NodeManager::currentNM();
   // Figure 3 : P(l), P(u), for s = 0,1
-  Node approx_b =
-      Rewriter::rewrite(approx.substitute(d_taylor.getTaylorVariable(), b));
+  // Node approx_b =
```

Comment:
Is this really not needed? If so delete.

Reply:
Did so, and moved the comment to where it now belongs.